packages/core/src/components/EmptyState/MissingAnnotationEmptyState.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Button, Typography } from '@material-ui/core';
+import { EmptyState } from './EmptyState';
+import PostAddIcon from '@material-ui/icons/PostAdd';
+import { useEntity } from '@backstage/plugin-catalog';
+import { CodeSnippet } from '../CodeSnippet';
+
+type ActionButtonProps = {
+  type: string;
+  target: string;
+};
+
+const ActionButton = ({ type, target }: ActionButtonProps) =>
+  type === 'github' ? (
+    <Button
+      variant="contained"
+      color="primary"
+      href={target.replace('blob', 'edit')}
```

Comment:
This very likely doesn't work as a general solution, prolly makes sense to add an `backstage.io/edit-location` annotation or something of that sort. Not this PR though

Reply:
Yep, maybe a naive approach from our side and we did wonder if it would hold up. So thanks for the input 👍